src/test/kotlin/dev/fakek/fakes/utils/FakeCompanyBaseTest.kt

```diff
@@ -0,0 +1,35 @@
+package dev.fakek.fakes.utils
+
+import dev.fakek.fakes.FakeCompany
+import dev.fakek.fakes.FakeUrl
+import io.mockk.every
+import io.mockk.mockk
+import dev.fakek.fakes.FakerCompany
+
+internal open class FakeCompanyBaseTest {
```

Comment:
I was thinking that, since ```FakeCommerce``` is a sub type of ```FakeCompany``` tests could also be sub types of each other